src/Microsoft.Extensions.ProjectModel.Sources/IProjectContext.cs

```diff
@@ -24,5 +25,7 @@ public interface IProjectContext
         IEnumerable<string> CompilationItems { get; }
         IEnumerable<string> EmbededItems { get; }
         string FindProperty(string propertyName);
+        IEnumerable<DependencyDescription> Dependencies { get;}
```

Comment:
These are Package Dependencies. 
The dependency description also provides information like what are its dependencies. 
Scaffolding has a scenario where it looks for all the projects package dependencies which in turn have a dependency on a well known package. 

Another additional information that is missing from the ResolvedReferences collection is the path to the Nuget package folder for a given package, which might be useful if looking for non compilation assets